src/dstack/_internal/server/schemas/projects.py

```diff
@@ -8,7 +8,11 @@
 
 class CreateProjectRequest(CoreModel):
     project_name: str
-    is_public: bool = False
+    is_public: Optional[bool] = False
```

Comment:
Should not be optional – unnecessary and leads to wrong type annotations downstream.